test/hotspot/jtreg/compiler/arguments/TestStressIGVNOptions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8252219
+ * @summary Tests that different combinations of the options -XX:+StressIGVN and
+ *          -XX:StressSeed=N are accepted.
+ * @run main/othervm -XX:+StressIGVN
+ *      compiler.arguments.TestStressIGVNOptions
+ * @run main/othervm -XX:+StressIGVN -XX:StressSeed=42
+ *      compiler.arguments.TestStressIGVNOptions
```

Comment:
Please, add next to run test when C2 is enabled as you did for other 2 tests:
@requires vm.compiler2.enabled

Reply:
Done, thanks for catching this up!